ontology/yaml/resources/HVAC/entity_types/HWS.yaml

```diff
@@ -313,6 +313,13 @@ HWS_SS_HWTTC_PWDPM2X:
   opt_uses:
   - compressor_run_status
 
+HWS_SWTC_HWVM:
+  description: Supply water temperature control with valve control
+  implements:
+  - HWS
+  - SWTC
+  - HWVM
```

Comment:
Change to `MXVPM` and add percentage sensor as opt_uses on the `MXVPM` type.

Reply:
Correction done.